app/uk/gov/hmrc/cataloguefrontend/connector/LeakDetectionConnector.scala

```diff
@@ -49,6 +50,20 @@ class LeakDetectionConnector @Inject() (
           Seq.empty
       }
   }
+
+  def leakDetectionRuleSummaries(implicit hc: HeaderCarrier): Future[Seq[LeakDetectionRuleSummary]] = {
+    implicit val ldrs = LeakDetectionRuleSummary.reads
+    http
+      .GET[Seq[LeakDetectionRuleSummary]](
+        url"$url/api/rule-summaries",
+        headers = Seq("Accept" -> "application/json")
+      )
+      .recover {
+        case NonFatal(ex) =>
+          logger.error(s"An error occurred when connecting to $url: ${ex.getMessage}", ex)
+          Seq.empty
+      }
```

Comment:
OBSERVATION: Does this mean we show a blank page if there's an error?

Reply:
Yeah, we just get the page title.  I've removed this now so we get the standard error page instead.